mtags-shared/src/main/scala/scala/meta/internal/mtags/CoursierComplete.scala

```diff
@@ -116,7 +116,7 @@ object CoursierComplete {
     (editStart, editEnd)
   }
 
-  val reg: Regex = """//>\s*using\s+(lib|plugin|dep)s?\s+"?(.*)""".r
```

Comment:
Won't there be a possibility to define custom scopes for `scala-cli`?

Reply:
Started a discussion on custom scopes in scala-cli: https://github.com/VirtusLab/scala-cli/discussions/3124